app/styles/_component-variables.scss

```diff
@@ -407,4 +407,19 @@ body {
   --notification-message-color-text-primary: var(--text-primary);
   --notification-dropdown-border-color: var(--neutral-grey-100);
   --notification-dropdown-arrow-border-color: var(--neutral-grey-200);
+
+  //variables for appmonitoring
+  --appmonitoring-font-weight-medium: var(--font-weight-medium);
+  --appmonitoring-color-black: var(--black);
+
+  //variables for appmonitoring/settings
+  --appmonitoring-settings-block-border-color: var(--neutral-white-100);
+  --appmonitoring-settings-block-description: var(--text-secondary-color);
```

Comment:
this should be `--text-secondary`

Reply:
It's `--text-secondary-color` on the design.

Reply:
both are same `--text-secondary` is standard css variable